apps/vs-code-designer/src/app/commands/createNewCodeProject/createCodeProjectSteps/createLogicApp/initLogicAppCodeProjectVScode/InitCodeProject.ts

```diff
@@ -0,0 +1,370 @@
+    await this.writeSettingsJson(context, vscodePath, language, version);
+    await this.writeExtensionsJson(context, vscodePath, language);
+
+    // Remove '.vscode' from gitignore if applicable
+    const gitignorePath: string = path.join(context.projectPath, '.gitignore');
+    if (await fse.pathExists(gitignorePath)) {
+      let gitignoreContents: string = (await fse.readFile(gitignorePath)).toString();
+      gitignoreContents = gitignoreContents.replace(/^\.vscode(\/|\\)?\s*$/gm, '');
+      await fse.writeFile(gitignorePath, gitignoreContents);
+    }
+  }
+
+  public shouldExecute(_context: IProjectWizardContext): boolean {
+    return true;
+  }
+
+  /**
+   * Overwrites the tasks.json file with the specified content.
+   * @param context The project wizard context.
+   **/
+  public async overwriteTasksJson(context: IProjectWizardContext): Promise<void> {
+    const tasksJsonPath: string = path.join(context.projectPath, '.vscode', 'Tasks.json');
+    const tasksJsonContent = `{
+    "version": "2.0.0",
+    "tasks": [
+      {
+        "label": "generateDebugSymbols",
+        "command": "dotnet",
+        "args": [
+          "\${input:getDebugSymbolDll}"
+        ],
+        "type": "process",
+        "problemMatcher": "$msCompile"
+      },
+      {
+        "type": "func",
+        "command": "host start",
+        "problemMatcher": "$func-watch",
+        "isBackground": true,
+        "label": "func: host start",
+        "group": {
+          "kind": "build",
+          "isDefault": true
+        }
+      }
+    ],
+    "inputs": [
+      {
+        "id": "getDebugSymbolDll",
+        "type": "command",
+        "command": "azureLogicAppsStandard.getDebugSymbolDll"
+      }
+    ]
+  }`;
+
+    if (await confirmOverwriteFile(context, tasksJsonPath)) {
+      await fse.writeFile(tasksJsonPath, tasksJsonContent);
+    }
+  }
+  protected setDeploySubpath(context: IProjectWizardContext, deploySubpath: string): string {
+    deploySubpath = this.addSubDir(context, deploySubpath);
+    this.settings.push({ key: deploySubpathSetting, value: deploySubpath });
+    return deploySubpath;
+  }
+
+  protected addSubDir(context: IProjectWizardContext, fsPath: string): string {
+    const subDir: string = path.relative(context.workspacePath, context.projectPath);
+    // always use posix for debug config
+    return path.posix.join(subDir, fsPath);
+  }
+
+  private async writeTasksJson(context: IProjectWizardContext, vscodePath: string): Promise<void> {
+    const newTasks: TaskDefinition[] = this.getTasks();
+    const versionMismatchError: Error = new Error(
+      localize('versionMismatchError', 'The version in your {0} must be "{1}" to work with Azure Functions.', tasksFileName, tasksVersion)
+    );
+
+    // Use VS Code api to update config if folder is open and it's not a multi-root workspace (https://github.com/Microsoft/vscode-azurefunctions/issues/1235)
+    // The VS Code api is better for several reasons, including:
```

Comment:
Same feedback as the other instance of this code comment:

```suggestion
    // The Visual Studio Code API is better for several reasons:
```